plugins/importexport/medra/classes/MedraWebservice.inc.php

```diff
@@ -149,6 +184,11 @@ function _doRequest($action, $arg, $attachment = null) {
 			if (empty($matches)) {
 				if ($attachment) {
 					assert(PKPString::regexp_match('#<returnCode>success</returnCode>#', $response));
+					if(!assert(PKPString::regexp_match('#<returnCode>success</returnCode>#', $response))){
+					    $parts = explode("\r\n\r\n", $response);
```

Comment:
Is the response always going to be divided by two blank lines in a row? (Also DOS-style line endings?)

Reply:
I used empty function instead of !assert